config/environments/test.rb

```diff
@@ -74,4 +74,7 @@
 
   config.akerdev_email = ENV.fetch('USER', 'user') + '@sanger.ac.uk'
 
+  config.ss_data_release_url = 'http://localhost:3000'
```

Comment:
Please could you change this to config.data_release_url (so it's consistent with the Projects app)